webapi/src/test/resources/test-data/resourcesR2RV2/Testding.jsonld

```diff
@@ -105,6 +105,52 @@
       "@value" : "http://0.0.0.0:3336/ark:/72163/1/0001/H6gBWUuJSuuO=CilHV8kQwk/bXMwnrHvQH2DMjOFrGmNzgz.20180528T155203897Z"
     }
   },
+  "anything:hasGeometry" : {
```

Comment:
I assume these values were introduced in #1514 but since the e2e tests were not run the difference in the test data was not detected.